import-export-cli/integration/devFirst_test.go

```diff
@@ -122,100 +126,131 @@ func TestInitializeAPIFromAPIDefinitionURL(t *testing.T) {
 		forceFlag: false,
 	}
 
-	output, err := initProjectWithOasFlag(t, args)
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	assert.Nil(t, err, "Error while generating Project")
-	assert.Contains(t, output, "Project initialized", "Test InitializeProjectWithDefinitionFlag Failed")
+	validateInitializeProjectWithOASFlag(t, args)
 
 	//Remove Created project and logout
 	base.RemoveDir(projectName)
```

Comment:
Can we move this _base.RemoveDir()_  into the _validateInitializeProjectWithOASFlag()_ function? It is an essential cleanup step when running this scenario but it is a low level detail that is not required to be known at the high level of the test case.